cylc/uiserver/resolvers.py

```diff
@@ -17,7 +17,10 @@
 """GraphQL resolvers for use in data accessing and mutation of workflows."""
 
 from fnmatch import fnmatchcase
```

Comment:
You removed some code block in this change that made this import irrelevant :+1: 

So ` from fnmatch import fnmatchcase` can be removed now.